owncloudApp/src/main/res/values-ar/strings.xml

```diff
@@ -487,8 +487,7 @@
   <string name="manage_space_clear_data">مسح البيانات</string>
   <string name="manage_space_error">تعذّر حذف بعض الملفات.</string>
   <string name="local_file_not_found_toast">لم يتم العثور على الملف في نظام الملفات المحلية</string>
-  <string name="confirmation_remove_files_alert">هل ترغب حقًا في إزالة العناصر المحددة؟</string>
-  <string name="confirmation_remove_folders_alert">هل ترغب حقًا في إزالة العناصر المحددة ومحتوياتها؟</string>
+    <string name="confirmation_remove_folders_alert">هل ترغب حقًا في إزالة العناصر المحددة ومحتوياتها؟</string>
```

Comment:
When you remove strings with the translator editor, some spaces are added in the line next to the last string that was removed. It's a bit boring, but we need to remove these spaces by hand so that Transifex doesn't complain in the future 😖

Reply:
I've removed all spaces before string tag.